fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordMetaDataBuilder.java

```diff
@@ -1179,6 +1191,14 @@ public void addFormerIndex(@Nonnull FormerIndex formerIndex) {
         formerIndexes.add(formerIndex);
     }
 
+    public void addUdf(@Nonnull Udf udf) {
+        udfMap.put(udf.getFunctionName(), udf);
+    }
+
+    public void addUdfs(@Nonnull Collection<Udf> udfs) {
```

Comment:
Make this an `Iterable<? extends Udf>`. That gives you the most generic type you actually need.

Reply:
done.